src/shared/ui/pixel-state.stories.tsx

```
@@ -0,0 +1,36 @@
+};
+
+export default meta;
+
+type Story = StoryObj<typeof PixelEmptyState>;
+
+export const Empty: Story = {
+  render: () => <PixelEmptyState message="아직 탐험 기록이 없습니다." />,
+};
+
+export const Error: Story = {
+  render: () => (
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)